test/test_basic.py

```diff
@@ -23,6 +23,8 @@ def test_basic_working():
         contents = index_file.read_text()
 
         assert 'Where we should see junk.a_key: has a value' in contents, f"docs/_data.junk.yaml does not have the right key/val junk.a_key"
+        assert "Which can also be written as junk['a_key']: has a value" in contents, f"docs/_data.junk.yaml does not have the right key/val junk.a_key"
```

Comment:
this is good but i meant more like; add your doc structure with the 1_file into the test structure at 
```
https://github.com/rosscdh/mkdocs-markdownextradata-plugin/tree/master/test/docs
```
so people can actually see a working example of your specific implementation

Reply:
You were faster than me :) I just submitted another change.

Reply:
beautiful mate! :) 💯